mods/d2k/rules/structures.yaml

```diff
@@ -100,18 +95,21 @@ CONCRETEB:
 		Type: Wood
 	RevealsShroud:
 		Range: 4c0
-	ProvidesCustomPrerequisite:
-		Prerequisite: power
+	RenderBuilding:
+		Image: power.harkonnen
+		RaceImages:
+			atreides: power.atreides
+			ordos: power.ordos
 	WithIdleOverlay@ZAPS:
 		Sequence: idle-zaps
 	Power:
 		Amount: 100
 	ScalePowerWithHealth:
 
-^BARRACKS:
+barr:
```

Comment:
Let's call this "barracks", please?
Also don't forget the map importer when you rename it. :)